python/cudf/io/csv.py

```diff
@@ -52,6 +57,16 @@ def read_csv(filepath, lineterminator='\n',
     doublequote : bool, default True
         When quotechar is specified and quoting is True, indicates whether to
         interpret two consecutive quotechar inside fields as single quotechar
+    header : int, default 'infer'
```

Comment:
Our default value isn't `infer` here, right? If so, we should update this docstring to reflect that and indicate the difference from Pandas.

Reply:
updated